src/libs/ifc/ifcdef.cpp

```diff
@@ -64,6 +64,7 @@
 #include <QString>
 
 #include "../vmisc/diagnostic.h"
+#include "qobject.h"
```

Comment:
Why this additional include?

Reply:
Do you mean #include "../vmisc/diagnostic.h"?... which is already included in ifcdef.h

![image](https://github.com/user-attachments/assets/58c511c7-d277-49b5-8116-e894bba2b963)

Reply:
This is another place where the DRS fork and the Seamly repo diverged. This issue was [fixed here](https://github.com/FashionFreedom/Seamly2D/pull/1086). I'll change this PR to implement the fix in that PR.